### PR TITLE
add tipGap

### PR DESCRIPTION
**Pull request by feilin-flexcompute**

### CODE REVIEW

**Comment by jbungenerflex** [comment]
please add a definition of how to calculate the non dimensional distance to the adjacent structure. 
Also please add a note that the structure must be effective at reducing blade tip vortices. I.e. being close to a fuselage or to another blade does not warrant playing with that number b/c I don't believe these affect the tip loss around the whole periphery. 
Apart from cowlings, what other structure would warrant playing with the tip loss factor?

**Review by maciej-flexcompute — DISMISSED** [review]

File: docs/source/examples/betDiskSteady_caseStudy.rst [hunk 1]
```diff
@@ -283,27 +283,27 @@ where :math:`R` is rotor disk radius and :math:`A` is rotor disk area, i.e. :mat
...
 
 In airplane mode, four blade collective angles are considered: :math:`26^o, 27^o, 28^o, 28.8^o` at :math:`r/R=0.75`. The flow conditions are:
 
 * Tip Mach Number = 0.54.
-* Reynolds Number (based on reference chord and tip speed, with no account for the advance velocity) = :math:`4.5\times 10^6`.
+* Reynolds Number (based on reference chord and tip speed, with no account for the inflow velocity) = :math:`4.5\times 10^6`.
 * Reference chord = 14 inch.
 * Reference Temperature= 288.15 K.
-* Advance ratio = 0.337
+* Advance ratio (defined as tip speed over inflow speed) = 0.337
```
Comment (jbungenerflex) [inline docs/source/examples/betDiskSteady_caseStudy.rst:297]: It's the other way around. Inflow speed/ tip speed.

File: docs/source/solverConfiguration/betTable.csv [hunk 2]
```diff
@@ -17,3 +17,4 @@ sectionalRadiuses@ Empty@ [list(float)] A list of the radial locations in grid u
 alphas@ Empty@ [list(float)] alphas associated with airfoil polars provided in *sectionalPolars* in degrees.
 MachNumbers@ Empty@ [list(float)] Mach numbers associated with airfoil polars provided in *sectionalPolars*.
 ReynoldsNumbers@ Empty@ [list(float)] Reynolds numbers associated with the airfoil polars provided in *sectionalPolars*.
+tipGap@ "inf"@ [float] non-dimensional distance between blade tip and peripharal structures, e.g. duct, shroud, cowling, nacelle, etc. The peripharal structures must be effective at reducing blade tip vortices. This parameter affects the tip loss effect. Being close to a fuselage or to another blade does not affect this parameter, because they won't reduce tip loss evidently. tipGap=0 means there is no tip loss. It is :math:`\infty` (default) for open propellers. An example with finite tipGap is `ducted fan <https://en.wikipedia.org/wiki/Ducted_fan>`_.
```
[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply docs/source/solverConfiguration/betTable.csv:20]: fixed.

**Review by jbungenerflex — APPROVED** [review]
Well done

**Pull request merged**